Bunkum.Protocols.Gemini/Bunkum.Protocols.Gemini.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Bunkum.Analyzers\Bunkum.Analyzers.csproj" />
+      <ProjectReference Include="..\Bunkum.Core\Bunkum.Core.csproj" />
+      <ProjectReference Include="..\Bunkum.Listener\Bunkum.Listener.csproj" />
+    </ItemGroup>
+
+</Project>
```

Comment:
This .csproj is missing NuGet information and a README, so it would probably be a bad idea to ship like this. You should also update the GHA workflow so this actually gets pushed.

Reply:
Im not sure what you mean about the csproj missing nuget information, neither Bunkum.Protocols.Gopher nor Bunkum.Protocols.Http contain extra info, although i will add it to the workflow file
![image](https://github.com/LittleBigRefresh/Bunkum/assets/34501060/df403c99-40c4-4355-8bc3-7ac8f3bbd084)